electricitymap/capacity_parsers/EMBER.py

```diff
@@ -0,0 +1,176 @@
+import io
+from datetime import datetime
+
+import pandas as pd
+import pycountry
+from bs4 import BeautifulSoup
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+""" Collects capacity data from the yearly electricity data from Ember"""
```

Comment:
Would it make sense to include the link to the readme file or documentation you are writing for the capacity updates?